config/locales/en.yml

```diff
@@ -40,8 +40,8 @@ en:
   find_an_event:
     types:
       222750001: |-
-        <p>Our Train to Teach events will provide you with a wealth of information and help you turn questions to confidence on your journey to the classroom.</p>
```

Comment:
@ethax-ross 

I'm presuming that this change should be revoked as:

1. this description is tied to an event type, and Get Into Teaching events have a new event type
2. I don't think this is used anyway - was this to do with the old event design?

Reply:
Yes, and yes probably, I'll take a look and remove it in the main PR if its redundant